reviewdog_test.go

```diff
@@ -47,7 +47,7 @@ golint.new.go:11:1: comment on exported function F2 should be of the form "F2 ..
 	p := NewErrorformatParser(efm)
 	c := NewRawCommentWriter(os.Stdout)
 	d := NewDiffString(difftext, 1)
-	app := NewReviewdog("tool name", p, c, d, FilterModeAdded)
+	app := NewReviewdog("tool name", p, c, d, FilterModeAdded, false)
```

Comment:
I can't see any tests which returns the exit code 1 and the message "input data has violations".
Can you write at least one test?

Reply:
I have added 2 tests that checks `nil` and `error` response based on passed option